x/emissions/types/keys.go

```diff
@@ -11,65 +11,70 @@ const (
+	DelegateStakeRemovalKey                           = collections.NewPrefix(22)
+	AllTopicStakeSumKey                               = collections.NewPrefix(23)
+	AddressTopicsKey                                  = collections.NewPrefix(24)
+	WhitelistAdminsKey                                = collections.NewPrefix(24)
+	ChurnableTopicsKey                                = collections.NewPrefix(25)
+	RewardableTopicsKey                               = collections.NewPrefix(26)
+	NetworkLossBundlesKey                             = collections.NewPrefix(27)
+	NetworkRegretsKey                                 = collections.NewPrefix(28)
+	StakeByReputerAndTopicIdKey                       = collections.NewPrefix(29)
+	ReputerScoresKey                                  = collections.NewPrefix(30)
+	InferenceScoresKey                                = collections.NewPrefix(31)
+	ForecastScoresKey                                 = collections.NewPrefix(32)
+	ReputerListeningCoefficientKey                    = collections.NewPrefix(33)
+	InfererNetworkRegretsKey                          = collections.NewPrefix(34)
+	ForecasterNetworkRegretsKey                       = collections.NewPrefix(35)
+	OneInForecasterNetworkRegretsKey                  = collections.NewPrefix(36)
+	UnfulfilledWorkerNoncesKey                        = collections.NewPrefix(37)
+	UnfulfilledReputerNoncesKey                       = collections.NewPrefix(38)
+	FeeRevenueEpochKey                                = collections.NewPrefix(39)
+	TopicFeeRevenueKey                                = collections.NewPrefix(40)
+	PreviousTopicWeightKey                            = collections.NewPrefix(41)
+	PreviousReputerRewardFractionKey                  = collections.NewPrefix(42)
+	PreviousInferenceRewardFractionKey                = collections.NewPrefix(43)
+	PreviousForecastRewardFractionKey                 = collections.NewPrefix(44)
+	LatestInfererScoresByWorkerKey                    = collections.NewPrefix(45)
+	LatestForecasterScoresByWorkerKey                 = collections.NewPrefix(46)
+	LatestReputerScoresByReputerKey                   = collections.NewPrefix(47)
+	TopicRewardNonceKey                               = collections.NewPrefix(48)
+	DelegateRewardPerShare                            = collections.NewPrefix(49)
+	PreviousPercentageRewardToStakedReputersKey       = collections.NewPrefix(50)
+	StakeRemovalsByBlockKey                           = collections.NewPrefix(51)
+	DelegateStakeRemovalsByBlockKey                   = collections.NewPrefix(52)
+	StakeRemovalsByActorKey                           = collections.NewPrefix(53)
+	DelegateStakeRemovalsByActorKey                   = collections.NewPrefix(54)
+	TopicLastWorkerCommitKey                          = collections.NewPrefix(55)
+	TopicLastReputerCommitKey                         = collections.NewPrefix(56)
+	TopicLastWorkerPayloadKey                         = collections.NewPrefix(57)
```

Comment:
I think this actually causes a problem for consensus on an upgrade from what I read somewhere if you change what keys are what so be VERY CAREFUL with this

I see these numbers have changed e.g. `TopicLastWorkerPayloadKey` is 57 here, but 58 elsewhere. I think we need to NOT be doing that since we're doing an upgrade. 

whatever thing you added, give it a number greater than the numbers we have, don't reassign numbers i'm like 80% confident that leads to consensus failures

Reply:
I didn't reassign numbers but good to know about this!